recipes/barrnap/0.3/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-mkdir -p $PREFIX
-mv ./* $PREFIX/
+mkdir -p $PREFIX/bin
+mv ./* $PREFIX/bin
```

Comment:
I think this could be dangerous, because it can flood bin with potentially non-executable files. Even if it is not the case right now, future releases might contain additional non-executables. I would suggest to explicitly copy the executables.